app/models/manageiq/providers/kubernetes/container_manager/container.rb

```diff
@@ -1,3 +1,3 @@
 class ManageIQ::Providers::Kubernetes::ContainerManager::Container < ::Container
-  delegate :pod_uid, :to => :container_group
+  has_one :pod_uid, through: :container_group
```

Comment:
No indeed, this was a way to expose a pod-specific attribute.

@kmjaszcz can you remove this change and update the patch?